src/main/java/useCases/Calculator/Calculator.java

```diff
@@ -0,0 +1,107 @@
+package useCases.Calculator;
```

Comment:
I like the use of the abstract class which includes attributes and methods inherited by all calculators